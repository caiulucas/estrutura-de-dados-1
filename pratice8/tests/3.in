5
1 102 2 62
1 128 3 127
1 144 4 80
1 102 4 101
2 62 3 61
2 100 4 80
2 88 5 82
3 79 4 90
3 87 5 100
4 110 5 99
3
1 30 2 10
1 10 3 30
2 40 3 10
3
1 30 2 0
1 10 3 0
2 40 3 10
3
2 40 3 10
1 30 2 10
1 10 3 30
4
1 50 2 0
1  0 3 25
1  0 4 25
2  0 3 25
2 50 4 0
3  0 4 150
10
1 97 2 20
1 50 3 70
1 51 4 86
1 49 5 91
1 87 6 76
1 27 7 20
1 130 8 64
1 20 9 103
1 31 10 11
2 24 3 41
2 3 4 50
2 110 5 129
2 66 6 3
2 42 7 14
2 87 8 33
2 53 9 28
2 97 10 53
3 9 4 102
3 62 5 18
3 93 6 57
3 77 7 18
3 82 8 119
3 92 9 76
3 86 10 101
4 12 5 122
4 42 6 110
4 108 7 14
4 17 8 22
4 35 9 43
4 76 10 59
5 87 6 122
5 45 7 128
5 100 8 53
5 71 9 53
5 110 10 32
6 49 7 34
6 23 8 56
6 1 9 2
6 101 10 114
7 106 8 87
7 66 9 112
7 126 10 17
8 39 9 44
8 86 10 13
9 71 10 73
30
1 35 2 129
1 90 3 18
1 30 4 3
1 23 5 83
1 72 6 40
1 89 7 5
1 74 8 6
1 70 9 125
1 105 10 111
1 86 11 71
1 26 12 37
1 26 13 14
1 25 14 28
1 86 15 66
1 31 16 126
1 124 17 114
1 2 18 65
1 67 19 83
1 36 20 31
1 70 21 90
1 95 22 108
1 113 23 28
1 22 24 38
1 19 25 53
1 124 26 127
1 34 27 105
1 119 28 20
1 48 29 60
1 125 30 13
2 9 3 3
2 116 4 25
2 90 5 2
2 85 6 117
2 17 7 27
2 116 8 121
2 98 9 86
2 114 10 80
2 118 11 81
2 4 12 6
2 3 13 7
2 111 14 127
2 17 15 36
2 95 16 99
2 111 17 64
2 66 18 90
2 115 19 119
2 121 20 51
2 38 21 74
2 100 22 76
2 66 23 54
2 10 24 85
2 35 25 34
2 114 26 124
2 129 27 22
2 29 28 117
2 114 29 1
2 37 30 9
3 107 4 130
3 64 5 1
3 91 6 88
3 77 7 129
3 50 8 75
3 18 9 68
3 13 10 87
3 11 11 117
3 72 12 79
3 112 13 21
3 14 14 106
3 128 15 96
3 83 16 13
3 13 17 26
3 34 18 67
3 66 19 49
3 50 20 11
3 99 21 129
3 128 22 11
3 85 23 45
3 113 24 47
3 4 25 102
3 89 26 125
3 74 27 14
3 34 28 30
3 6 29 56
3 22 30 48
4 60 5 3
4 29 6 104
4 41 7 72
4 121 8 63
4 74 9 106
4 105 10 41
4 51 11 42
4 87 12 103
4 20 13 5
4 107 14 69
4 64 15 23
4 36 16 66
4 95 17 7
4 63 18 70
4 117 19 101
4 104 20 84
4 58 21 46
4 118 22 2
4 65 23 98
4 74 24 108
4 19 25 8
4 49 26 49
4 22 27 69
4 74 28 6
4 74 29 41
4 63 30 50
5 116 6 7
5 14 7 99
5 39 8 80
5 51 9 77
5 31 10 25
5 71 11 24
5 26 12 89
5 57 13 58
5 36 14 90
5 98 15 1
5 49 16 54
5 123 17 16
5 21 18 70
5 111 19 66
5 116 20 95
5 102 21 43
5 11 22 101
5 51 23 116
5 62 24 49
5 74 25 101
5 125 26 92
5 51 27 14
5 72 28 20
5 110 29 107
5 107 30 108
6 31 7 77
6 93 8 26
6 96 9 24
6 89 10 114
6 79 11 76
6 118 12 75
6 45 13 50
6 35 14 129
6 48 15 96
6 67 16 97
6 59 17 122
6 6 18 61
6 81 19 109
6 85 20 77
6 54 21 61
6 8 22 61
6 87 23 85
6 108 24 100
6 84 25 52
6 127 26 67
6 11 27 32
6 82 28 115
6 114 29 56
6 21 30 117
7 83 8 31
7 23 9 88
7 18 10 11
7 119 11 28
7 105 12 99
7 29 13 74
7 5 14 28
7 113 15 36
7 6 16 91
7 13 17 90
7 26 18 89
7 121 19 10
7 124 20 37
7 92 21 73
7 59 22 108
7 9 23 113
7 70 24 11
7 22 25 31
7 58 26 88
7 56 27 11
7 85 28 32
7 60 29 84
7 120 30 89
8 50 9 42
8 2 10 125
8 84 11 62
8 72 12 28
8 64 13 75
8 17 14 66
8 43 15 26
8 8 16 75
8 86 17 51
8 82 18 78
8 35 19 108
8 118 20 10
8 42 21 91
8 44 22 73
8 31 23 102
8 13 24 33
8 28 25 81
8 13 26 15
8 42 27 111
8 55 28 84
8 19 29 106
8 1 30 71
9 16 10 62
9 113 11 9
9 87 12 102
9 79 13 64
9 74 14 121
9 81 15 67
9 9 16 115
9 86 17 125
9 28 18 40
9 120 19 99
9 113 20 55
9 36 21 3
9 86 22 25
9 1 23 91
9 32 24 105
9 36 25 2
9 10 26 47
9 19 27 19
9 82 28 96
9 87 29 97
9 34 30 26
10 10 11 37
10 32 12 43
10 82 13 96
10 64 14 59
10 114 15 72
10 75 16 47
10 72 17 19
10 109 18 30
10 5 19 72
10 73 20 10
10 57 21 41
10 59 22 83
10 49 23 75
10 42 24 11
10 36 25 5
10 42 26 75
10 118 27 45
10 11 28 73
10 2 29 70
10 12 30 74
11 121 12 115
11 4 13 86
11 116 14 62
11 3 15 113
11 123 16 120
11 30 17 76
11 28 18 50
11 125 19 89
11 99 20 76
11 81 21 36
11 111 22 4
11 49 23 122
11 65 24 98
11 38 25 59
11 3 26 66
11 51 27 49
11 4 28 123
11 54 29 54
11 37 30 120
12 109 13 57
12 3 14 30
12 79 15 9
12 97 16 30
12 106 17 73
12 109 18 66
12 70 19 56
12 48 20 89
12 57 21 118
12 47 22 112
12 47 23 94
12 13 24 49
12 41 25 97
12 21 26 17
12 6 27 95
12 21 28 57
12 87 29 115
12 123 30 23
13 53 14 35
13 108 15 90
13 26 16 29
13 85 17 86
13 45 18 95
13 82 19 2
13 113 20 102
13 66 21 128
13 46 22 29
13 126 23 78
13 94 24 87
13 51 25 16
13 73 26 100
13 84 27 72
13 95 28 29
13 64 29 77
13 36 30 18
14 46 15 41
14 127 16 61
14 25 17 130
14 1 18 42
14 13 19 107
14 105 20 113
14 12 21 78
14 26 22 20
14 107 23 7
14 22 24 120
14 90 25 27
14 99 26 95
14 123 27 43
14 119 28 64
14 81 29 57
14 97 30 25
15 86 16 127
15 126 17 94
15 5 18 111
15 87 19 127
15 110 20 17
15 95 21 61
15 81 22 121
15 128 23 121
15 111 24 58
15 84 25 20
15 114 26 70
15 112 27 53
15 116 28 107
15 33 29 101
15 126 30 66
16 63 17 130
16 93 18 81
16 61 19 58
16 55 20 98
16 115 21 18
16 79 22 34
16 25 23 80
16 70 24 30
16 87 25 23
16 42 26 51
16 120 27 41
16 93 28 26
16 2 29 102
16 73 30 79
17 15 18 35
17 35 19 68
17 18 20 77
17 5 21 127
17 95 22 79
17 97 23 59
17 93 24 79
17 28 25 45
17 75 26 118
17 10 27 98
17 18 28 31
17 71 29 52
17 77 30 8
18 110 19 34
18 113 20 79
18 114 21 53
18 120 22 127
18 74 23 18
18 15 24 8
18 86 25 78
18 7 26 109
18 57 27 53
18 97 28 100
18 88 29 85
18 52 30 41
19 72 20 98
19 19 21 70
19 78 22 13
19 46 23 96
19 45 24 58
19 110 25 29
19 25 26 29
19 46 27 99
19 107 28 99
19 46 29 60
19 39 30 63
20 115 21 53
20 23 22 96
20 50 23 82
20 122 24 110
20 77 25 102
20 41 26 64
20 76 27 96
20 62 28 119
20 46 29 122
20 20 30 106
21 4 22 25
21 124 23 45
21 13 24 50
21 110 25 100
21 32 26 59
21 111 27 18
21 113 28 16
21 97 29 3
21 113 30 32
22 3 23 89
22 23 24 59
22 24 25 44
22 32 26 98
22 89 27 86
22 61 28 78
22 102 29 109
22 23 30 65
23 115 24 95
23 65 25 35
23 93 26 94
23 111 27 96
23 112 28 74
23 77 29 94
23 126 30 79
24 38 25 59
24 118 26 128
24 42 27 60
24 27 28 11
24 96 29 74
24 21 30 116
25 94 26 27
25 92 27 123
25 88 28 116
25 21 29 76
25 39 30 22
26 118 27 114
26 57 28 20
26 113 29 100
26 48 30 4
27 62 28 108
27 106 29 85
27 15 30 49
28 60 29 18
28 92 30 41
29 27 30 26
30
1 103 2 76
1 42 3 2
1 59 4 73
1 81 5 78
1 68 6 130
1 14 7 7
1 2 8 60
1 124 9 111
1 4 10 8
1 69 11 81
1 107 12 64
1 86 13 129
1 58 14 31
1 9 15 100
1 71 16 17
1 92 17 117
1 119 18 44
1 116 19 4
1 82 20 47
1 46 21 67
1 74 22 19
1 79 23 60
1 40 24 76
1 83 25 72
1 23 26 44
1 107 27 21
1 20 28 129
1 30 29 62
1 32 30 77
2 93 3 38
2 25 4 103
2 16 5 55
2 59 6 13
2 60 7 2
2 69 8 10
2 28 9 106
2 35 10 13
2 88 11 106
2 48 12 75
2 118 13 41
2 62 14 70
2 69 15 94
2 26 16 81
2 28 17 98
2 6 18 58
2 30 19 121
2 46 20 30
2 43 21 46
2 48 22 104
2 113 23 103
2 78 24 117
2 129 25 11
2 117 26 113
2 57 27 87
2 127 28 34
2 104 29 45
2 9 30 58
3 9 4 42
3 9 5 34
3 91 6 36
3 27 7 14
3 44 8 121
3 36 9 72
3 45 10 87
3 59 11 84
3 70 12 28
3 39 13 6
3 118 14 68
3 24 15 25
3 59 16 45
3 90 17 20
3 78 18 32
3 74 19 98
3 2 20 86
3 122 21 83
3 97 22 93
3 83 23 18
3 90 24 10
3 97 25 119
3 72 26 4
3 32 27 25
3 30 28 11
3 79 29 70
3 95 30 18
4 63 5 103
4 123 6 23
4 55 7 22
4 120 8 70
4 26 9 129
4 81 10 121
4 83 11 17
4 35 12 47
4 57 13 35
4 24 14 124
4 128 15 23
4 48 16 95
4 106 17 30
4 99 18 78
4 95 19 54
4 27 20 63
4 86 21 28
4 49 22 19
4 88 23 11
4 9 24 38
4 29 25 114
4 130 26 90
4 6 27 111
4 16 28 35
4 29 29 63
4 86 30 39
5 3 6 27
5 56 7 3
5 80 8 109
5 33 9 25
5 87 10 45
5 72 11 59
5 77 12 43
5 53 13 121
5 29 14 35
5 18 15 62
5 21 16 57
5 37 17 18
5 52 18 27
5 90 19 53
5 91 20 81
5 107 21 45
5 48 22 94
5 72 23 33
5 57 24 127
5 41 25 104
5 33 26 14
5 57 27 113
5 103 28 109
5 13 29 110
5 41 30 1
6 58 7 30
6 47 8 62
6 88 9 95
6 17 10 99
6 50 11 47
6 92 12 107
6 71 13 26
6 59 14 9
6 6 15 12
6 116 16 115
6 129 17 46
6 29 18 19
6 127 19 55
6 34 20 1
6 2 21 10
6 40 22 75
6 6 23 60
6 24 24 87
6 56 25 94
6 11 26 40
6 17 27 105
6 130 28 102
6 111 29 87
6 99 30 59
7 44 8 116
7 32 9 85
7 103 10 42
7 96 11 60
7 61 12 99
7 109 13 130
7 74 14 63
7 122 15 18
7 104 16 79
7 43 17 15
7 54 18 29
7 4 19 53
7 24 20 71
7 27 21 4
7 62 22 5
7 120 23 125
7 80 24 105
7 16 25 21
7 81 26 52
7 21 27 112
7 111 28 11
7 73 29 129
7 17 30 55
8 4 9 64
8 78 10 120
8 19 11 46
8 98 12 2
8 72 13 22
8 25 14 122
8 127 15 99
8 93 16 87
8 61 17 116
8 7 18 42
8 94 19 77
8 58 20 87
8 97 21 114
8 112 22 39
8 94 23 40
8 103 24 128
8 118 25 97
8 13 26 51
8 53 27 7
8 29 28 111
8 103 29 124
8 92 30 54
9 10 10 99
9 84 11 55
9 97 12 70
9 17 13 90
9 46 14 60
9 44 15 74
9 113 16 13
9 52 17 26
9 23 18 76
9 42 19 25
9 75 20 11
9 18 21 54
9 35 22 128
9 121 23 46
9 99 24 7
9 105 25 83
9 7 26 109
9 49 27 58
9 18 28 104
9 34 29 65
9 9 30 64
10 76 11 77
10 102 12 121
10 66 13 128
10 23 14 125
10 6 15 107
10 31 16 97
10 94 17 23
10 68 18 65
10 72 19 85
10 37 20 37
10 16 21 46
10 104 22 44
10 18 23 64
10 129 24 121
10 55 25 51
10 128 26 7
10 127 27 1
10 128 28 99
10 94 29 62
10 39 30 20
11 117 12 99
11 121 13 69
11 4 14 81
11 36 15 59
11 96 16 75
11 121 17 73
11 117 18 111
11 45 19 94
11 85 20 4
11 55 21 43
11 49 22 9
11 9 23 52
11 21 24 45
11 106 25 6
11 26 26 115
11 83 27 14
11 83 28 13
11 93 29 74
11 3 30 87
12 32 13 129
12 71 14 98
12 78 15 22
12 115 16 57
12 61 17 122
12 35 18 70
12 78 19 115
12 37 20 83
12 127 21 86
12 92 22 58
12 42 23 103
12 117 24 118
12 130 25 125
12 68 26 69
12 26 27 93
12 91 28 70
12 38 29 57
12 79 30 32
13 89 14 116
13 108 15 64
13 3 16 19
13 4 17 12
13 94 18 80
13 36 19 40
13 97 20 90
13 63 21 127
13 114 22 8
13 3 23 49
13 118 24 114
13 77 25 70
13 10 26 13
13 70 27 38
13 69 28 48
13 33 29 18
13 81 30 27
14 46 15 10
14 21 16 83
14 33 17 49
14 88 18 115
14 74 19 68
14 64 20 55
14 63 21 7
14 55 22 48
14 32 23 65
14 5 24 42
14 54 25 108
14 15 26 14
14 61 27 123
14 11 28 83
14 110 29 94
14 103 30 91
15 44 16 25
15 73 17 124
15 108 18 76
15 13 19 31
15 85 20 52
15 58 21 76
15 124 22 17
15 82 23 112
15 23 24 25
15 2 25 86
15 99 26 77
15 70 27 17
15 99 28 30
15 123 29 80
15 41 30 79
16 103 17 96
16 89 18 84
16 29 19 46
16 77 20 66
16 117 21 41
16 117 22 31
16 48 23 45
16 26 24 110
16 5 25 130
16 85 26 49
16 126 27 7
16 23 28 54
16 84 29 65
16 14 30 122
17 70 18 76
17 42 19 54
17 8 20 43
17 89 21 130
17 66 22 36
17 76 23 35
17 66 24 53
17 97 25 63
17 43 26 114
17 113 27 123
17 41 28 47
17 54 29 67
17 121 30 37
18 101 19 76
18 68 20 74
18 20 21 115
18 39 22 8
18 50 23 61
18 61 24 46
18 81 25 9
18 44 26 126
18 49 27 27
18 90 28 109
18 93 29 16
18 8 30 2
19 49 20 75
19 12 21 49
19 85 22 102
19 48 23 2
19 76 24 56
19 40 25 115
19 122 26 95
19 25 27 79
19 124 28 42
19 51 29 85
19 81 30 75
20 102 21 94
20 73 22 130
20 15 23 61
20 63 24 35
20 110 25 22
20 71 26 111
20 82 27 121
20 123 28 26
20 81 29 129
20 114 30 68
21 32 22 121
21 69 23 105
21 17 24 57
21 12 25 62
21 7 26 67
21 30 27 92
21 92 28 108
21 39 29 102
21 6 30 106
22 128 23 101
22 81 24 116
22 106 25 69
22 94 26 33
22 32 27 99
22 36 28 44
22 34 29 15
22 120 30 68
23 125 24 102
23 34 25 6
23 73 26 6
23 98 27 40
23 18 28 102
23 74 29 59
23 34 30 56
24 26 25 80
24 65 26 32
24 100 27 106
24 9 28 41
24 9 29 63
24 107 30 40
25 54 26 45
25 113 27 10
25 112 28 44
25 50 29 107
25 112 30 16
26 56 27 122
26 94 28 79
26 7 29 73
26 128 30 37
27 117 28 41
27 72 29 23
27 129 30 51
28 92 29 42
28 104 30 7
29 46 30 100
0
