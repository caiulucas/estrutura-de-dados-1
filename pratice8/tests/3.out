Instancia 1
1 2 4 5 3

Instancia 2
2 1 3

Instancia 3
1 2 3

Instancia 4
2 1 3

Instancia 5
4 3 1 2

Instancia 6
5 3 4 9 2 1 8 6 7 10

Instancia 7
26 18 14 16 13 23 4 7 29 21 2 10 28 12 9 5 1 24 15 25 17 30 3 27 6 22 19 11 8 20

Instancia 8
27 3 4 14 11 13 10 24 15 23 5 25 9 8 1 16 28 19 29 20 6 18 26 2 17 22 12 7 21 30
